packages/kbn-legacy-logging/BUILD.bazel

```diff
@@ -0,0 +1,92 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config", "ts_project")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library", "pkg_npm")
+
+PKG_BASE_NAME = "kbn-legacy-logging"
+PKG_REQUIRE_NAME = "@kbn/legacy-logging"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+  ],
+  exclude = ["**/*.test.*"],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+  "README.md"
+]
+
+SRC_DEPS = [
+  "//packages/kbn-config-schema",
+  "@npm//chokidar",
+  "@npm//@elastic/numeral",
+  "@npm//@hapi/hapi",
+  "@npm//lodash",
+  "@npm//moment-timezone",
+  "@npm//query-string",
+  "@npm//rxjs",
+  "@npm//tslib",
+]
+
+TYPES_DEPS = [
+  "@npm//@types/hapi__hapi",
+  "@npm//@types/hapi__podium",
+  "@npm//@types/jest",
+  "@npm//@types/lodash",
+  "@npm//@types/moment-timezone",
+  "@npm//@types/node",
+]
+
+DEPS = SRC_DEPS + TYPES_DEPS
+
+ts_config(
+  name = "tsconfig",
+  src = "tsconfig.json",
+  deps = [
+    "//:tsconfig.base.json",
+  ],
+)
+
+ts_project(
+  name = "tsc",
+  args = ['--pretty'],
+  srcs = SRCS,
+  deps = DEPS,
+  declaration = True,
+  declaration_map = True,
+  incremental = False,
```

Comment:
Can we enable the incremental mode ? It was disable before, but now on bazel it can be enabled

Reply:
Pushed with 668af7d